packages/sources/superstate/src/endpoint/nav.ts

```diff
@@ -0,0 +1,26 @@
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { config } from '../config'
+import { NavTransport } from '../transport/nav'
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+
+export const inputParameters = new InputParameters({
+  fundId: {
+    description: 'Fund id',
+    type: 'number',
+    default: 1,
```

Comment:
Minor: Does it make sense to default the fund id? Could this cause a mis-formed request to think they're requesting  a specific fund but actually getting back fund id `1`?